client-v2/src/util/frontsUtils.ts

```diff
@@ -43,9 +47,26 @@ const isFrontStale = (lastUpdated?: number, lastPressed?: number) => {
   return false;
 };
 
+const getVisibilityStoryDetails = (groupsWithStories: ArticleFragment[][]) => groupsWithStories.reduce((stories, storiesInGroup, index) => {
```

Comment:
I think I need to understand stories a bit more - not quite sure what this is doing ...

Reply:
Fear not, stories are just badly named articles. This has been fixed now.